problems.rb

```diff
@@ -0,0 +1,9 @@
+#TODO: fix this whole fuction
+def hithisfuncissocomp!!licated(n)
+    for i in 0..n
+        for j in 0..3
+            puts 'HI'
```

Comment:
This definitely works but it would better store “Hi” in a variable to avoid magic values.